Tests/TestingTests/PlanTests.swift

```diff
@@ -27,7 +30,9 @@ struct PlanTests {
     ]
 
     var configuration = Configuration()
-    configuration.selectedTestIDs = [innerTestType.id]
+    configuration.testSelectionFilter = { test in
+      Test.ID.Selection(testIDs: [innerTestType.id]).contains(test)
```

Comment:
This is exponentially complex to compute. Recommend moving the creation of the `Selection` instance out of the closure.

Reply:
Done!